docker-compose.yml

```diff
@@ -10,11 +10,20 @@ services:
       - MYSQL_HOST=mysql
       - MYSQL_PORT=3306
       - MYSQL_DATABASE=activitypub
+      - NODE_ENV=testing
+      - ALLOW_PRIVATE_ADDRESS=true
+      - SKIP_SIGNATURE_VERIFICATION=true
```

Comment:
Should we make a note here (comment) or / and track (in linear task) that we need to come back to this once the signature stuff has been sorted (or maybe make a note on the task for that if we have one?)